### PR TITLE
Prevent creation of channels with empty names

### LINKED ISSUE #32: Add new channels empty o with blank spaces
When you create a new channel to the list, it allows you to add channel empty or with blank spaces.

### PR DESCRIPTION
**Pull request by diegotc86**
Close #32 

It prevents the creation of channels with empty names (blank or empty string)

### CODE REVIEW

File: public/assets/js/index.js [hunk 1]
```diff
@@ -64,7 +66,7 @@ function initializeConnection() {
 }
 
 function connectionSocket() {
-  socket = new WebSocket("ws://192.168.86.81:3000");
+  socket = new WebSocket("ws://192.168.86.55:3000");
```
Comment (Linzeur) [inline public/assets/js/index.js]: We need to discuss to define it later, and what it will be the the final IP

File: public/assets/js/index.js [hunk 2]
```diff
@@ -98,10 +100,18 @@ function handleAddChannelSubmit(event) {
...
+  if (result == 1) {
     listChannels();
     $newChannelName.value = "";
-  } else $error.innerText = "This channel already exists";
+  } else if (result == 0) {
+    console.log("entro a 0");
+    $error.innerText = "This channel already exists";
+  } else {
+    $error.innerText = "Invalid channelname";
+  }
```
Comment (Linzeur) [inline public/assets/js/index.js]: I see that you had to change the type of data to return on base to the conditionals. You did good!

File: public/assets/js/index.js [hunk 3]
```diff
@@ -25,12 +25,14 @@ function createNewMessage(message, author, typeMessage) {
 }
 
 function addChannel(name, author) {
+  if (name.trim().length == 0) return -1;
```
Comment (Gatts1) [inline public/assets/js/index.js]: What does trim method do?

Comment (diegotc86) [reply public/assets/js/index.js]: Trim method erase blank spaces at the begging and end of a string.
`"     hi     ".trim() = "hi"`

**Pull request merged**